Tests/SentryTests/SentryTransactionTests.swift

```diff
@@ -110,4 +110,34 @@ class SentryTransactionTest: XCTestCase {
         XCTAssertNotNil((serialization["contexts"] as! Dictionary)["trace"])
         XCTAssertNotNil(serialization["spans"])
     }
+    
+    func testAdditionOfChild() {
+        let transaction = Transaction(name: someTransactionName)
+        transaction.startChild(operation: someOperation)
+        XCTAssertEqual(transaction.spans.count, 1)
+    }
+    
+    func testSerializeWithSpan() {
+        let transaction = Transaction(name: someTransactionName)
+        transaction.startChild(operation: someOperation)
+        
+        let serialization = transaction.serialize()
+        let spansSerialized = serialization["spans"] as! [String: Any]
+        XCTAssertEqual(spansSerialized.count, 1)
+    }
+    
+    func testAddChildWithOperation() {
+        let transaction = Transaction(name: someTransactionName)
+        let span = transaction.startChild(operation: someOperation)
+        XCTAssertEqual(span.operation, someOperation)
```

Comment:
`l`: It would be nice to also make sure `sampled` is `true` when it is `true` in the transaction, so we know it is passed along correctly. Otherwise I could do `span.sampled = NO;` in `startChildWithParentId` and the test would still pass.
```suggestion
        XCTAssertEqual(span.operation, someOperation)
        XCTAssertNil(span.spanDescription)
        XCTAssertFalse(span.sampled)
```